custom_components/unifiprotect/__init__.py

```diff
@@ -56,7 +56,7 @@ def _async_import_options_from_data_if_missing(hass: HomeAssistant, entry: Confi
         hass.config_entries.async_update_entry(entry, data=data, options=options)
 
 
-async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
```

Comment:
```suggestion
async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

This is expected to return True/False

Its only the platforms that don't need it